aiohttp/client_reqrep.py

```diff
@@ -209,6 +210,7 @@ def __init__(self, method, url, *,
         if data or self.method not in self.GET_METHODS:
             self.update_transfer_encoding()
         self.update_expect_continue(expect100)
+        self._traces = traces or []
```

Comment:
In other aiohttp code we usually use another pattern:
```
if traces is None:
    traces = []
self._traces = traces
```
Please change the one-liner to explicit check.
It is a very subtle optimization for if `traces` is an empty list already you code calculates `False` for `bool(traces)` check and creates an empty list again.

Reply:
done